src/grid/atomgrid.py

```diff
@@ -234,7 +244,7 @@ def l_max(self):
         """int: Largest angular degree L value in angular grids."""
         return np.max(self._rad_degs)
 
-    def get_shell_grid(self, index, r_sq=True):
+    def get_shell_grid(self, index, r_sq=True):  # reason
```

Comment:
I think this was a question for you to add the reason for defining this function no? In any event, the `# reason` comment should be removed.